src/modules/md/md.cpp

```diff
@@ -48,6 +49,17 @@ MDModule::MDModule() : Module("MD")
         "Only forces arising from intramolecular terms (including pair potential contributions) will be calculated",
         intramolecularForcesOnly_);
 
+    // Output
+    keywords_.add<OptionalIntegerKeyword>("Output", "EnergyFrequency",
+                                          "Frequency at which to calculate total system energy (or 0 to inhibit)",
```

Comment:
I think we can now remove the "(or 0 to inhibit)" from the help text and the comments above the variables in the header.
```suggestion
                                          "Frequency at which to calculate total system energy",
```